Source/IntegrationTests/TestFiles/LitTests/LitTest/git-issues/git-issue-2959.dfy

```diff
@@ -1,4 +1,4 @@
-// RUN: %exits-with 3 %baredafny build --show-snippets:false --use-basename-for-filename --enforce-determinism "%s" > "%t"
+// RUN: ! %baredafny resolve --show-snippets:false --use-basename-for-filename --enforce-determinism "%s" > "%t"
```

Comment:
The exit codes we have don't seem useful to me. If the tool is used by a human, than it's better to look at the stdout, which shows diagnostics messages.

I'd prefer to ignore exit code since I don't think they've been designed well. If we would use exit codes, I think they should be picked so an automated tool can make decisions based on them such as "retry the invocation", "an internal error was detected, send diagnostics output to the Dafny team"